Reports the number of times the best tree score is reached.

In [1]:
import numpy as np
import pandas as pd
from matplotlib import pyplot as plt
import seaborn as sns;
from collections import Counter
import glob
from tqdm import tqdm_notebook as tqdm
import os 
%matplotlib inline
sns.set(rc={'figure.figsize':(9.7,4.27)})

In [2]:
output_path = "/cluster/work/bewi/members/tuncel/sc-dna/notebooks/manuscript_simulations_max_tree_reproducibility/"
all_paths = glob.glob("/cluster/work/bewi/members/tuncel/data/dna/manuscript_/iterative_simulations_segmented_threshold_2/*")
scores_df = pd.DataFrame()
for path in tqdm(sorted(all_paths)): # 27
    #import ipdb; ipdb.set_trace()
    dups_per_sim = []
    for sim_id in range(0,100): # 100
        trees_in_folder_in_sim = glob.glob(path+'/'+str(sim_id)+'_*_tree_inferred.txt')
        tree_scores = [] 
        for tree_file in trees_in_folder_in_sim: # 10
            tree_score = np.genfromtxt(tree_file)[0][2]
            tree_scores.append(tree_score)
        max_ts = max(tree_scores)
        max_tree_scores = [ts for ts in tree_scores if ts==max_ts]
        n_dups = len(max_tree_scores) - len(set(max_tree_scores))
        dups_per_sim.append(n_dups)

    scores_df[path.split('/')[-1]] = dups_per_sim
#scores_df.to_csv(output_path+"determining_n_iters_segmented_512_coeff" + '.csv')

HBox(children=(IntProgress(value=0, max=27), HTML(value='')))

In [3]:
pd.set_option('display.max_columns', None)  # or 1000
pd.set_option('display.max_rows', None)  # or 1000
pd.set_option('display.max_colwidth', -1)  # or 199

In [4]:
scores_df.shape

(100, 27)

In [5]:
scores_df

10nodes_10regions_100000reads  10nodes_10regions_10000reads  \
0   9                              9                              
1   9                              7                              
2   5                              3                              
3   1                              3                              
4   9                              1                              
5   9                              8                              
6   0                              0                              
7   8                              3                              
8   5                              7                              
9   9                              7                              
10  9                              5                              
11  8                              0                              
12  8                              0                              
13  9                              4                              
14  9                              6                              
15  9                              1                              
16  9                              7                              
17  9                              1                              
18  9                              0                              
19  9                              0                              
20  9                              9                              
21  9                              0                              
22  8                              5                              
23  9                              6                              
24  9                              5                              
25  8                              9                              
26  9                              2                              
27  5                              4                              
28  9                              5                              
29  7                              5                              
30  9                              7                              
31  9                              6                              
32  9                              4                              
33  8                              7                              
34  9                              5                              
35  9                              0                              
36  8                              0                              
37  6                              4                              
38  8                              1                              
39  6                              6                              
40  9                              3                              
41  9                              7                              
42  8                              3                              
43  7                              3                              
44  9                              1                              
45  9                              7                              
46  8                              7                              
47  8                              8                              
48  6                              3                              
49  4                              0                              
50  8                              6                              
51  9                              6                              
52  8                              0                              
53  9                              0                              
54  9                              7                              
55  9                              5                              
56  9                              7                              
57  9                              8                              
58  7                              4               

In [9]:
scores_df.min(axis=0)

10nodes_10regions_100000reads     0
10nodes_10regions_10000reads      0
10nodes_10regions_30000reads      0
10nodes_20regions_100000reads     0
10nodes_20regions_10000reads      0
10nodes_20regions_30000reads      0
10nodes_40regions_100000reads     0
10nodes_40regions_10000reads      0
10nodes_40regions_30000reads      0
20nodes_20regions_100000reads     3
20nodes_20regions_10000reads      0
20nodes_20regions_30000reads      0
20nodes_40regions_100000reads     0
20nodes_40regions_10000reads      0
20nodes_40regions_30000reads      0
20nodes_80regions_100000reads     0
20nodes_80regions_10000reads      0
20nodes_80regions_30000reads      0
30nodes_120regions_100000reads    0
30nodes_120regions_10000reads     0
30nodes_120regions_30000reads     0
30nodes_30regions_100000reads     0
30nodes_30regions_10000reads      0
30nodes_30regions_30000reads      0
30nodes_60regions_100000reads     0
30nodes_60regions_10000reads      0
30nodes_60regions_30000reads      0
dtype: int64

In [7]:
scores_df.mean()

10nodes_10regions_100000reads     8.00
10nodes_10regions_10000reads      4.53
10nodes_10regions_30000reads      6.33
10nodes_20regions_100000reads     8.21
10nodes_20regions_10000reads      3.86
10nodes_20regions_30000reads      6.60
10nodes_40regions_100000reads     8.16
10nodes_40regions_10000reads      4.39
10nodes_40regions_30000reads      7.03
20nodes_20regions_100000reads     8.37
20nodes_20regions_10000reads      0.00
20nodes_20regions_30000reads      1.38
20nodes_40regions_100000reads     7.87
20nodes_40regions_10000reads      0.09
20nodes_40regions_30000reads      2.02
20nodes_80regions_100000reads     7.77
20nodes_80regions_10000reads      0.12
20nodes_80regions_30000reads      2.45
30nodes_120regions_100000reads    7.44
30nodes_120regions_10000reads     0.18
30nodes_120regions_30000reads     1.48
30nodes_30regions_100000reads     8.33
30nodes_30regions_10000reads      0.02
30nodes_30regions_30000reads      2.21
30nodes_60regions_100000reads     7.99
30nodes_60regions_10000re

In [10]:
42

42

### Renaming the failed ones

In [12]:
scores_df_cols = scores_df.columns.values

In [13]:
scores_df_cols

array(['10nodes_10regions_100000reads', '10nodes_10regions_10000reads',
       '10nodes_10regions_30000reads', '10nodes_20regions_100000reads',
       '10nodes_20regions_10000reads', '10nodes_20regions_30000reads',
       '10nodes_40regions_100000reads', '10nodes_40regions_10000reads',
       '10nodes_40regions_30000reads', '20nodes_20regions_100000reads',
       '20nodes_20regions_10000reads', '20nodes_20regions_30000reads',
       '20nodes_40regions_100000reads', '20nodes_40regions_10000reads',
       '20nodes_40regions_30000reads', '20nodes_80regions_100000reads',
       '20nodes_80regions_10000reads', '20nodes_80regions_30000reads',
       '30nodes_120regions_100000reads', '30nodes_120regions_10000reads',
       '30nodes_120regions_30000reads', '30nodes_30regions_100000reads',
       '30nodes_30regions_10000reads', '30nodes_30regions_30000reads',
       '30nodes_60regions_100000reads', '30nodes_60regions_10000reads',
       '30nodes_60regions_30000reads'], dtype=object)

In [14]:
cols_10_nodes = [col for col in scores_df_cols if col.startswith('10nodes')]

In [15]:
cols_10_nodes

['10nodes_10regions_100000reads',
 '10nodes_10regions_10000reads',
 '10nodes_10regions_30000reads',
 '10nodes_20regions_100000reads',
 '10nodes_20regions_10000reads',
 '10nodes_20regions_30000reads',
 '10nodes_40regions_100000reads',
 '10nodes_40regions_10000reads',
 '10nodes_40regions_30000reads']

In [19]:
cols_20_nodes = [col for col in scores_df_cols if col.startswith('20nodes')]
cols_20_nodes

['20nodes_20regions_100000reads',
 '20nodes_20regions_10000reads',
 '20nodes_20regions_30000reads',
 '20nodes_40regions_100000reads',
 '20nodes_40regions_10000reads',
 '20nodes_40regions_30000reads',
 '20nodes_80regions_100000reads',
 '20nodes_80regions_10000reads',
 '20nodes_80regions_30000reads']

In [20]:
cols_30_nodes = [col for col in scores_df_cols if col.startswith('30nodes')]
cols_30_nodes

['30nodes_120regions_100000reads',
 '30nodes_120regions_10000reads',
 '30nodes_120regions_30000reads',
 '30nodes_30regions_100000reads',
 '30nodes_30regions_10000reads',
 '30nodes_30regions_30000reads',
 '30nodes_60regions_100000reads',
 '30nodes_60regions_10000reads',
 '30nodes_60regions_30000reads']

In [23]:
for col in cols_30_nodes:
    file_path = "/cluster/work/bewi/members/tuncel/data/dna/manuscript_/iterative_simulations_segmented_threshold_2/"+col+'/'
    values_to_replace = scores_df.index[scores_df[col]<5].values
    print(col)
    print(values_to_replace)
    for value in values_to_replace:
        for filename in os.listdir(file_path):
            if filename.startswith(str(value)+"_"):
                print(filename)
                #new_name = ".rep6_"+filename
                #os.remove(file_path+filename)

30nodes_120regions_100000reads
[30 50 58 92]
30_infrep4_inferred_cnvs.txt
30_infrep0_inferred_cnvs.txt
30_infrep8_inferred_cnvs.txt
30_infrep4_tree_inferred.txt
30_infrep2_tree_inferred.txt
30_infrep1_inferred_cnvs.txt
30_infrep0_tree_inferred.txt
30_infrep7_inferred_cnvs.txt
30_infrep1_tree_inferred.txt
30_infrep6_inferred_cnvs.txt
30_infrep5_inferred_cnvs.txt
30_infrep5_tree_inferred.txt
30_infrep3_inferred_cnvs.txt
30_infrep3_tree_inferred.txt
30_infrep8_tree_inferred.txt
30_infrep6_tree_inferred.txt
30_infrep7_tree_inferred.txt
30_infrep9_inferred_cnvs.txt
30_infrep2_inferred_cnvs.txt
30_infrep9_tree_inferred.txt
50_infrep6_inferred_cnvs.txt
50_infrep3_inferred_cnvs.txt
50_infrep0_tree_inferred.txt
50_infrep0_inferred_cnvs.txt
50_infrep4_inferred_cnvs.txt
50_infrep1_inferred_cnvs.txt
50_infrep5_inferred_cnvs.txt
50_infrep2_inferred_cnvs.txt
50_infrep3_tree_inferred.txt
50_infrep9_inferred_cnvs.txt
50_infrep5_tree_inferred.txt
50_infrep4_tree_inferred.txt
50_infrep2_tree_inferred.tx

14_infrep9_inferred_cnvs.txt
15_infrep8_tree_inferred.txt
15_infrep2_tree_inferred.txt
15_infrep1_inferred_cnvs.txt
15_infrep9_tree_inferred.txt
15_infrep0_inferred_cnvs.txt
15_infrep5_tree_inferred.txt
15_infrep0_tree_inferred.txt
15_infrep7_inferred_cnvs.txt
15_infrep1_tree_inferred.txt
15_infrep8_inferred_cnvs.txt
15_infrep6_tree_inferred.txt
15_infrep7_tree_inferred.txt
15_infrep6_inferred_cnvs.txt
15_infrep4_tree_inferred.txt
15_infrep2_inferred_cnvs.txt
15_infrep9_inferred_cnvs.txt
15_infrep3_tree_inferred.txt
15_infrep5_inferred_cnvs.txt
15_infrep3_inferred_cnvs.txt
15_infrep4_inferred_cnvs.txt
16_infrep4_tree_inferred.txt
16_infrep6_inferred_cnvs.txt
16_infrep7_inferred_cnvs.txt
16_infrep9_inferred_cnvs.txt
16_infrep8_tree_inferred.txt
16_infrep1_tree_inferred.txt
16_infrep3_inferred_cnvs.txt
16_infrep3_tree_inferred.txt
16_infrep1_inferred_cnvs.txt
16_infrep4_inferred_cnvs.txt
16_infrep6_tree_inferred.txt
16_infrep5_tree_inferred.txt
16_infrep9_tree_inferred.txt
16_infrep7_tre

34_infrep3_tree_inferred.txt
34_infrep2_inferred_cnvs.txt
34_infrep4_inferred_cnvs.txt
34_infrep8_tree_inferred.txt
34_infrep1_inferred_cnvs.txt
34_infrep3_inferred_cnvs.txt
34_infrep0_tree_inferred.txt
34_infrep9_tree_inferred.txt
34_infrep0_inferred_cnvs.txt
34_infrep7_tree_inferred.txt
34_infrep5_inferred_cnvs.txt
34_infrep5_tree_inferred.txt
34_infrep4_tree_inferred.txt
34_infrep9_inferred_cnvs.txt
34_infrep1_tree_inferred.txt
34_infrep6_inferred_cnvs.txt
34_infrep8_inferred_cnvs.txt
35_infrep6_inferred_cnvs.txt
35_infrep9_inferred_cnvs.txt
35_infrep0_tree_inferred.txt
35_infrep3_inferred_cnvs.txt
35_infrep5_tree_inferred.txt
35_infrep0_inferred_cnvs.txt
35_infrep9_tree_inferred.txt
35_infrep2_inferred_cnvs.txt
35_infrep3_tree_inferred.txt
35_infrep1_inferred_cnvs.txt
35_infrep6_tree_inferred.txt
35_infrep7_inferred_cnvs.txt
35_infrep4_tree_inferred.txt
35_infrep5_inferred_cnvs.txt
35_infrep7_tree_inferred.txt
35_infrep8_tree_inferred.txt
35_infrep1_tree_inferred.txt
35_infrep4_inf

56_infrep0_tree_inferred.txt
56_infrep6_inferred_cnvs.txt
56_infrep5_inferred_cnvs.txt
56_infrep9_tree_inferred.txt
56_infrep9_inferred_cnvs.txt
56_infrep8_tree_inferred.txt
56_infrep1_inferred_cnvs.txt
56_infrep7_tree_inferred.txt
56_infrep5_tree_inferred.txt
56_infrep6_tree_inferred.txt
56_infrep2_tree_inferred.txt
56_infrep2_inferred_cnvs.txt
56_infrep4_tree_inferred.txt
56_infrep3_inferred_cnvs.txt
56_infrep0_inferred_cnvs.txt
56_infrep4_inferred_cnvs.txt
56_infrep7_inferred_cnvs.txt
56_infrep1_tree_inferred.txt
56_infrep3_tree_inferred.txt
56_infrep8_inferred_cnvs.txt
57_infrep0_inferred_cnvs.txt
57_infrep7_inferred_cnvs.txt
57_infrep5_inferred_cnvs.txt
57_infrep2_inferred_cnvs.txt
57_infrep8_inferred_cnvs.txt
57_infrep1_inferred_cnvs.txt
57_infrep2_tree_inferred.txt
57_infrep6_tree_inferred.txt
57_infrep6_inferred_cnvs.txt
57_infrep3_tree_inferred.txt
57_infrep9_tree_inferred.txt
57_infrep0_tree_inferred.txt
57_infrep4_tree_inferred.txt
57_infrep3_inferred_cnvs.txt
57_infrep7_tre

83_infrep9_inferred_cnvs.txt
83_infrep4_tree_inferred.txt
83_infrep9_tree_inferred.txt
83_infrep3_inferred_cnvs.txt
83_infrep7_inferred_cnvs.txt
83_infrep1_tree_inferred.txt
83_infrep0_inferred_cnvs.txt
83_infrep5_tree_inferred.txt
83_infrep7_tree_inferred.txt
83_infrep8_inferred_cnvs.txt
83_infrep0_tree_inferred.txt
83_infrep4_inferred_cnvs.txt
84_infrep9_tree_inferred.txt
84_infrep6_inferred_cnvs.txt
84_infrep2_inferred_cnvs.txt
84_infrep5_tree_inferred.txt
84_infrep5_inferred_cnvs.txt
84_infrep1_tree_inferred.txt
84_infrep8_tree_inferred.txt
84_infrep2_tree_inferred.txt
84_infrep0_inferred_cnvs.txt
84_infrep6_tree_inferred.txt
84_infrep7_inferred_cnvs.txt
84_infrep1_inferred_cnvs.txt
84_infrep3_inferred_cnvs.txt
84_infrep4_tree_inferred.txt
84_infrep8_inferred_cnvs.txt
84_infrep3_tree_inferred.txt
84_infrep7_tree_inferred.txt
84_infrep9_inferred_cnvs.txt
84_infrep0_tree_inferred.txt
84_infrep4_inferred_cnvs.txt
86_infrep8_inferred_cnvs.txt
86_infrep5_tree_inferred.txt
86_infrep7_tre

7_infrep2_inferred_cnvs.txt
7_infrep3_inferred_cnvs.txt
7_infrep4_inferred_cnvs.txt
7_infrep5_inferred_cnvs.txt
7_infrep9_tree_inferred.txt
7_infrep7_inferred_cnvs.txt
7_infrep5_tree_inferred.txt
7_infrep1_tree_inferred.txt
7_infrep0_tree_inferred.txt
7_infrep4_tree_inferred.txt
7_infrep6_inferred_cnvs.txt
8_infrep2_tree_inferred.txt
8_infrep6_tree_inferred.txt
8_infrep3_tree_inferred.txt
8_infrep5_tree_inferred.txt
8_infrep4_tree_inferred.txt
8_infrep5_inferred_cnvs.txt
8_infrep7_inferred_cnvs.txt
8_infrep8_inferred_cnvs.txt
8_infrep0_inferred_cnvs.txt
8_infrep4_inferred_cnvs.txt
8_infrep1_inferred_cnvs.txt
8_infrep6_inferred_cnvs.txt
8_infrep2_inferred_cnvs.txt
8_infrep0_tree_inferred.txt
8_infrep3_inferred_cnvs.txt
8_infrep7_tree_inferred.txt
8_infrep8_tree_inferred.txt
8_infrep1_tree_inferred.txt
8_infrep9_tree_inferred.txt
8_infrep9_inferred_cnvs.txt
9_infrep6_inferred_cnvs.txt
9_infrep9_tree_inferred.txt
9_infrep6_tree_inferred.txt
9_infrep4_tree_inferred.txt
9_infrep4_inferred_c

23_infrep3_tree_inferred.txt
23_infrep7_inferred_cnvs.txt
23_infrep5_inferred_cnvs.txt
23_infrep7_tree_inferred.txt
23_infrep9_tree_inferred.txt
23_infrep1_tree_inferred.txt
24_infrep6_inferred_cnvs.txt
24_infrep7_inferred_cnvs.txt
24_infrep9_inferred_cnvs.txt
24_infrep4_tree_inferred.txt
24_infrep5_tree_inferred.txt
24_infrep0_tree_inferred.txt
24_infrep8_tree_inferred.txt
24_infrep1_tree_inferred.txt
24_infrep2_tree_inferred.txt
24_infrep5_inferred_cnvs.txt
24_infrep9_tree_inferred.txt
24_infrep6_tree_inferred.txt
24_infrep4_inferred_cnvs.txt
24_infrep3_tree_inferred.txt
24_infrep3_inferred_cnvs.txt
24_infrep8_inferred_cnvs.txt
24_infrep2_inferred_cnvs.txt
24_infrep1_inferred_cnvs.txt
24_infrep7_tree_inferred.txt
24_infrep0_inferred_cnvs.txt
25_infrep1_inferred_cnvs.txt
25_infrep6_tree_inferred.txt
25_infrep5_inferred_cnvs.txt
25_infrep4_inferred_cnvs.txt
25_infrep3_inferred_cnvs.txt
25_infrep2_tree_inferred.txt
25_infrep4_tree_inferred.txt
25_infrep2_inferred_cnvs.txt
25_infrep3_tre

48_infrep0_tree_inferred.txt
48_infrep4_inferred_cnvs.txt
48_infrep6_tree_inferred.txt
48_infrep7_tree_inferred.txt
48_infrep2_tree_inferred.txt
48_infrep5_tree_inferred.txt
48_infrep3_inferred_cnvs.txt
48_infrep5_inferred_cnvs.txt
48_infrep9_inferred_cnvs.txt
48_infrep4_tree_inferred.txt
48_infrep9_tree_inferred.txt
48_infrep1_inferred_cnvs.txt
49_infrep0_inferred_cnvs.txt
49_infrep4_inferred_cnvs.txt
49_infrep2_inferred_cnvs.txt
49_infrep3_tree_inferred.txt
49_infrep3_inferred_cnvs.txt
49_infrep5_tree_inferred.txt
49_infrep0_tree_inferred.txt
49_infrep1_inferred_cnvs.txt
49_infrep5_inferred_cnvs.txt
49_infrep7_inferred_cnvs.txt
49_infrep1_tree_inferred.txt
49_infrep6_inferred_cnvs.txt
49_infrep4_tree_inferred.txt
49_infrep7_tree_inferred.txt
49_infrep2_tree_inferred.txt
49_infrep9_tree_inferred.txt
49_infrep6_tree_inferred.txt
49_infrep9_inferred_cnvs.txt
49_infrep8_inferred_cnvs.txt
49_infrep8_tree_inferred.txt
50_infrep6_inferred_cnvs.txt
50_infrep3_inferred_cnvs.txt
50_infrep0_tre

71_infrep7_tree_inferred.txt
71_infrep9_tree_inferred.txt
72_infrep5_inferred_cnvs.txt
72_infrep3_inferred_cnvs.txt
72_infrep4_tree_inferred.txt
72_infrep0_tree_inferred.txt
72_infrep6_tree_inferred.txt
72_infrep7_inferred_cnvs.txt
72_infrep9_inferred_cnvs.txt
72_infrep3_tree_inferred.txt
72_infrep0_inferred_cnvs.txt
72_infrep5_tree_inferred.txt
72_infrep6_inferred_cnvs.txt
72_infrep1_inferred_cnvs.txt
72_infrep7_tree_inferred.txt
72_infrep2_tree_inferred.txt
72_infrep2_inferred_cnvs.txt
72_infrep8_tree_inferred.txt
72_infrep1_tree_inferred.txt
72_infrep4_inferred_cnvs.txt
72_infrep9_tree_inferred.txt
72_infrep8_inferred_cnvs.txt
73_infrep4_tree_inferred.txt
73_infrep8_tree_inferred.txt
73_infrep1_inferred_cnvs.txt
73_infrep3_tree_inferred.txt
73_infrep5_tree_inferred.txt
73_infrep9_inferred_cnvs.txt
73_infrep8_inferred_cnvs.txt
73_infrep7_tree_inferred.txt
73_infrep0_inferred_cnvs.txt
73_infrep2_inferred_cnvs.txt
73_infrep0_tree_inferred.txt
73_infrep6_tree_inferred.txt
73_infrep5_inf

0_infrep2_inferred_cnvs.txt
0_infrep8_tree_inferred.txt
0_infrep1_tree_inferred.txt
0_infrep0_tree_inferred.txt
0_infrep3_inferred_cnvs.txt
0_infrep4_tree_inferred.txt
0_infrep8_inferred_cnvs.txt
0_infrep7_tree_inferred.txt
0_infrep4_inferred_cnvs.txt
0_infrep5_tree_inferred.txt
0_infrep6_inferred_cnvs.txt
0_infrep1_inferred_cnvs.txt
0_infrep6_tree_inferred.txt
0_infrep7_inferred_cnvs.txt
0_infrep5_inferred_cnvs.txt
0_infrep3_tree_inferred.txt
1_infrep3_inferred_cnvs.txt
1_infrep4_tree_inferred.txt
1_infrep1_tree_inferred.txt
1_infrep1_inferred_cnvs.txt
1_infrep7_tree_inferred.txt
1_infrep5_tree_inferred.txt
1_infrep3_tree_inferred.txt
1_infrep0_inferred_cnvs.txt
1_infrep8_inferred_cnvs.txt
1_infrep9_tree_inferred.txt
1_infrep6_inferred_cnvs.txt
1_infrep7_inferred_cnvs.txt
1_infrep2_tree_inferred.txt
1_infrep9_inferred_cnvs.txt
1_infrep0_tree_inferred.txt
1_infrep8_tree_inferred.txt
1_infrep6_tree_inferred.txt
1_infrep5_inferred_cnvs.txt
1_infrep4_inferred_cnvs.txt
1_infrep2_inferred_c

20_infrep0_tree_inferred.txt
20_infrep5_tree_inferred.txt
20_infrep9_inferred_cnvs.txt
20_infrep4_tree_inferred.txt
20_infrep6_tree_inferred.txt
20_infrep6_inferred_cnvs.txt
20_infrep8_inferred_cnvs.txt
20_infrep1_inferred_cnvs.txt
20_infrep1_tree_inferred.txt
20_infrep0_inferred_cnvs.txt
20_infrep9_tree_inferred.txt
20_infrep4_inferred_cnvs.txt
20_infrep7_tree_inferred.txt
20_infrep8_tree_inferred.txt
20_infrep2_tree_inferred.txt
20_infrep3_tree_inferred.txt
20_infrep2_inferred_cnvs.txt
20_infrep5_inferred_cnvs.txt
20_infrep3_inferred_cnvs.txt
20_infrep7_inferred_cnvs.txt
21_infrep6_inferred_cnvs.txt
21_infrep0_tree_inferred.txt
21_infrep3_inferred_cnvs.txt
21_infrep2_inferred_cnvs.txt
21_infrep3_tree_inferred.txt
21_infrep6_tree_inferred.txt
21_infrep9_tree_inferred.txt
21_infrep0_inferred_cnvs.txt
21_infrep7_tree_inferred.txt
21_infrep2_tree_inferred.txt
21_infrep1_inferred_cnvs.txt
21_infrep8_tree_inferred.txt
21_infrep7_inferred_cnvs.txt
21_infrep9_inferred_cnvs.txt
21_infrep4_tre

39_infrep1_tree_inferred.txt
39_infrep1_inferred_cnvs.txt
39_infrep2_inferred_cnvs.txt
39_infrep6_tree_inferred.txt
39_infrep7_inferred_cnvs.txt
39_infrep7_tree_inferred.txt
39_infrep8_inferred_cnvs.txt
39_infrep3_inferred_cnvs.txt
40_infrep1_tree_inferred.txt
40_infrep5_tree_inferred.txt
40_infrep6_inferred_cnvs.txt
40_infrep3_tree_inferred.txt
40_infrep3_inferred_cnvs.txt
40_infrep5_inferred_cnvs.txt
40_infrep2_tree_inferred.txt
40_infrep7_tree_inferred.txt
40_infrep9_inferred_cnvs.txt
40_infrep4_tree_inferred.txt
40_infrep8_inferred_cnvs.txt
40_infrep8_tree_inferred.txt
40_infrep2_inferred_cnvs.txt
40_infrep0_tree_inferred.txt
40_infrep6_tree_inferred.txt
40_infrep1_inferred_cnvs.txt
40_infrep0_inferred_cnvs.txt
40_infrep9_tree_inferred.txt
40_infrep4_inferred_cnvs.txt
40_infrep7_inferred_cnvs.txt
41_infrep7_tree_inferred.txt
41_infrep2_inferred_cnvs.txt
41_infrep1_tree_inferred.txt
41_infrep3_tree_inferred.txt
41_infrep6_tree_inferred.txt
41_infrep4_inferred_cnvs.txt
41_infrep6_inf

63_infrep2_inferred_cnvs.txt
63_infrep7_inferred_cnvs.txt
63_infrep6_tree_inferred.txt
63_infrep7_tree_inferred.txt
63_infrep0_inferred_cnvs.txt
63_infrep4_tree_inferred.txt
63_infrep0_tree_inferred.txt
63_infrep4_inferred_cnvs.txt
63_infrep8_inferred_cnvs.txt
63_infrep1_inferred_cnvs.txt
63_infrep5_tree_inferred.txt
63_infrep3_inferred_cnvs.txt
63_infrep1_tree_inferred.txt
63_infrep3_tree_inferred.txt
63_infrep9_inferred_cnvs.txt
64_infrep3_tree_inferred.txt
64_infrep0_tree_inferred.txt
64_infrep6_tree_inferred.txt
64_infrep2_inferred_cnvs.txt
64_infrep8_inferred_cnvs.txt
64_infrep1_inferred_cnvs.txt
64_infrep5_inferred_cnvs.txt
64_infrep8_tree_inferred.txt
64_infrep7_inferred_cnvs.txt
64_infrep9_tree_inferred.txt
64_infrep9_inferred_cnvs.txt
64_infrep7_tree_inferred.txt
64_infrep0_inferred_cnvs.txt
64_infrep1_tree_inferred.txt
64_infrep6_inferred_cnvs.txt
64_infrep4_inferred_cnvs.txt
64_infrep2_tree_inferred.txt
64_infrep5_tree_inferred.txt
64_infrep4_tree_inferred.txt
64_infrep3_inf

89_infrep0_inferred_cnvs.txt
89_infrep6_inferred_cnvs.txt
89_infrep1_inferred_cnvs.txt
89_infrep9_inferred_cnvs.txt
89_infrep8_tree_inferred.txt
89_infrep4_inferred_cnvs.txt
89_infrep9_tree_inferred.txt
89_infrep6_tree_inferred.txt
89_infrep8_inferred_cnvs.txt
89_infrep7_inferred_cnvs.txt
89_infrep2_inferred_cnvs.txt
89_infrep0_tree_inferred.txt
89_infrep5_tree_inferred.txt
89_infrep1_tree_inferred.txt
89_infrep3_inferred_cnvs.txt
89_infrep7_tree_inferred.txt
89_infrep2_tree_inferred.txt
89_infrep3_tree_inferred.txt
89_infrep4_tree_inferred.txt
90_infrep5_tree_inferred.txt
90_infrep9_inferred_cnvs.txt
90_infrep5_inferred_cnvs.txt
90_infrep7_inferred_cnvs.txt
90_infrep1_tree_inferred.txt
90_infrep7_tree_inferred.txt
90_infrep4_tree_inferred.txt
90_infrep6_inferred_cnvs.txt
90_infrep8_inferred_cnvs.txt
90_infrep9_tree_inferred.txt
90_infrep3_inferred_cnvs.txt
90_infrep2_tree_inferred.txt
90_infrep2_inferred_cnvs.txt
90_infrep4_inferred_cnvs.txt
90_infrep0_tree_inferred.txt
90_infrep3_tre

14_infrep5_tree_inferred.txt
14_infrep1_tree_inferred.txt
14_infrep8_inferred_cnvs.txt
14_infrep0_inferred_cnvs.txt
14_infrep7_tree_inferred.txt
14_infrep9_tree_inferred.txt
14_infrep7_inferred_cnvs.txt
14_infrep8_tree_inferred.txt
14_infrep2_tree_inferred.txt
14_infrep5_inferred_cnvs.txt
14_infrep6_tree_inferred.txt
14_infrep4_tree_inferred.txt
14_infrep1_inferred_cnvs.txt
14_infrep0_tree_inferred.txt
14_infrep2_inferred_cnvs.txt
14_infrep3_inferred_cnvs.txt
14_infrep4_inferred_cnvs.txt
14_infrep3_tree_inferred.txt
14_infrep6_inferred_cnvs.txt
14_infrep9_inferred_cnvs.txt
16_infrep4_tree_inferred.txt
16_infrep6_inferred_cnvs.txt
16_infrep7_inferred_cnvs.txt
16_infrep9_inferred_cnvs.txt
16_infrep8_tree_inferred.txt
16_infrep1_tree_inferred.txt
16_infrep3_inferred_cnvs.txt
16_infrep3_tree_inferred.txt
16_infrep1_inferred_cnvs.txt
16_infrep4_inferred_cnvs.txt
16_infrep6_tree_inferred.txt
16_infrep5_tree_inferred.txt
16_infrep9_tree_inferred.txt
16_infrep7_tree_inferred.txt
16_infrep8_inf

38_infrep2_inferred_cnvs.txt
38_infrep4_inferred_cnvs.txt
38_infrep3_inferred_cnvs.txt
38_infrep1_tree_inferred.txt
38_infrep0_tree_inferred.txt
38_infrep8_tree_inferred.txt
38_infrep9_inferred_cnvs.txt
38_infrep4_tree_inferred.txt
39_infrep4_tree_inferred.txt
39_infrep0_inferred_cnvs.txt
39_infrep5_inferred_cnvs.txt
39_infrep9_inferred_cnvs.txt
39_infrep9_tree_inferred.txt
39_infrep8_tree_inferred.txt
39_infrep5_tree_inferred.txt
39_infrep0_tree_inferred.txt
39_infrep4_inferred_cnvs.txt
39_infrep2_tree_inferred.txt
39_infrep6_inferred_cnvs.txt
39_infrep3_tree_inferred.txt
39_infrep1_tree_inferred.txt
39_infrep1_inferred_cnvs.txt
39_infrep2_inferred_cnvs.txt
39_infrep6_tree_inferred.txt
39_infrep7_inferred_cnvs.txt
39_infrep7_tree_inferred.txt
39_infrep8_inferred_cnvs.txt
39_infrep3_inferred_cnvs.txt
40_infrep1_tree_inferred.txt
40_infrep5_tree_inferred.txt
40_infrep6_inferred_cnvs.txt
40_infrep3_tree_inferred.txt
40_infrep3_inferred_cnvs.txt
40_infrep5_inferred_cnvs.txt
40_infrep2_tre

62_infrep1_tree_inferred.txt
62_infrep6_tree_inferred.txt
62_infrep9_tree_inferred.txt
62_infrep9_inferred_cnvs.txt
62_infrep2_inferred_cnvs.txt
62_infrep5_tree_inferred.txt
62_infrep0_tree_inferred.txt
62_infrep4_tree_inferred.txt
62_infrep0_inferred_cnvs.txt
62_infrep2_tree_inferred.txt
62_infrep7_tree_inferred.txt
62_infrep5_inferred_cnvs.txt
62_infrep1_inferred_cnvs.txt
62_infrep4_inferred_cnvs.txt
62_infrep3_tree_inferred.txt
62_infrep6_inferred_cnvs.txt
62_infrep8_inferred_cnvs.txt
62_infrep3_inferred_cnvs.txt
62_infrep7_inferred_cnvs.txt
62_infrep8_tree_inferred.txt
64_infrep3_tree_inferred.txt
64_infrep0_tree_inferred.txt
64_infrep6_tree_inferred.txt
64_infrep2_inferred_cnvs.txt
64_infrep8_inferred_cnvs.txt
64_infrep1_inferred_cnvs.txt
64_infrep5_inferred_cnvs.txt
64_infrep8_tree_inferred.txt
64_infrep7_inferred_cnvs.txt
64_infrep9_tree_inferred.txt
64_infrep9_inferred_cnvs.txt
64_infrep7_tree_inferred.txt
64_infrep0_inferred_cnvs.txt
64_infrep1_tree_inferred.txt
64_infrep6_inf

99_infrep7_tree_inferred.txt
99_infrep7_inferred_cnvs.txt
99_infrep0_tree_inferred.txt
99_infrep3_inferred_cnvs.txt
99_infrep6_tree_inferred.txt
99_infrep8_tree_inferred.txt
99_infrep6_inferred_cnvs.txt
99_infrep4_inferred_cnvs.txt
99_infrep9_tree_inferred.txt
99_infrep5_tree_inferred.txt
99_infrep2_inferred_cnvs.txt
99_infrep2_tree_inferred.txt
99_infrep5_inferred_cnvs.txt
99_infrep3_tree_inferred.txt
99_infrep8_inferred_cnvs.txt
99_infrep1_inferred_cnvs.txt
99_infrep9_inferred_cnvs.txt
99_infrep4_tree_inferred.txt
99_infrep0_inferred_cnvs.txt
99_infrep1_tree_inferred.txt
30nodes_60regions_100000reads
[10 35 80]
10_infrep8_tree_inferred.txt
10_infrep4_tree_inferred.txt
10_infrep0_inferred_cnvs.txt
10_infrep0_tree_inferred.txt
10_infrep6_inferred_cnvs.txt
10_infrep5_inferred_cnvs.txt
10_infrep5_tree_inferred.txt
10_infrep7_inferred_cnvs.txt
10_infrep2_inferred_cnvs.txt
10_infrep7_tree_inferred.txt
10_infrep2_tree_inferred.txt
10_infrep9_tree_inferred.txt
10_infrep3_tree_inferred.txt
10

14_infrep5_inferred_cnvs.txt
14_infrep6_tree_inferred.txt
14_infrep4_tree_inferred.txt
14_infrep1_inferred_cnvs.txt
14_infrep0_tree_inferred.txt
14_infrep2_inferred_cnvs.txt
14_infrep3_inferred_cnvs.txt
14_infrep4_inferred_cnvs.txt
14_infrep3_tree_inferred.txt
14_infrep6_inferred_cnvs.txt
14_infrep9_inferred_cnvs.txt
15_infrep8_tree_inferred.txt
15_infrep2_tree_inferred.txt
15_infrep1_inferred_cnvs.txt
15_infrep9_tree_inferred.txt
15_infrep0_inferred_cnvs.txt
15_infrep5_tree_inferred.txt
15_infrep0_tree_inferred.txt
15_infrep7_inferred_cnvs.txt
15_infrep1_tree_inferred.txt
15_infrep8_inferred_cnvs.txt
15_infrep6_tree_inferred.txt
15_infrep7_tree_inferred.txt
15_infrep6_inferred_cnvs.txt
15_infrep4_tree_inferred.txt
15_infrep2_inferred_cnvs.txt
15_infrep9_inferred_cnvs.txt
15_infrep3_tree_inferred.txt
15_infrep5_inferred_cnvs.txt
15_infrep3_inferred_cnvs.txt
15_infrep4_inferred_cnvs.txt
16_infrep4_tree_inferred.txt
16_infrep6_inferred_cnvs.txt
16_infrep7_inferred_cnvs.txt
16_infrep9_inf

34_infrep6_tree_inferred.txt
34_infrep2_tree_inferred.txt
34_infrep3_tree_inferred.txt
34_infrep2_inferred_cnvs.txt
34_infrep4_inferred_cnvs.txt
34_infrep8_tree_inferred.txt
34_infrep1_inferred_cnvs.txt
34_infrep3_inferred_cnvs.txt
34_infrep0_tree_inferred.txt
34_infrep9_tree_inferred.txt
34_infrep0_inferred_cnvs.txt
34_infrep7_tree_inferred.txt
34_infrep5_inferred_cnvs.txt
34_infrep5_tree_inferred.txt
34_infrep4_tree_inferred.txt
34_infrep9_inferred_cnvs.txt
34_infrep1_tree_inferred.txt
34_infrep6_inferred_cnvs.txt
34_infrep8_inferred_cnvs.txt
35_infrep6_inferred_cnvs.txt
35_infrep9_inferred_cnvs.txt
35_infrep0_tree_inferred.txt
35_infrep3_inferred_cnvs.txt
35_infrep5_tree_inferred.txt
35_infrep0_inferred_cnvs.txt
35_infrep9_tree_inferred.txt
35_infrep2_inferred_cnvs.txt
35_infrep3_tree_inferred.txt
35_infrep1_inferred_cnvs.txt
35_infrep6_tree_inferred.txt
35_infrep7_inferred_cnvs.txt
35_infrep4_tree_inferred.txt
35_infrep5_inferred_cnvs.txt
35_infrep7_tree_inferred.txt
35_infrep8_tre

56_infrep2_tree_inferred.txt
56_infrep2_inferred_cnvs.txt
56_infrep4_tree_inferred.txt
56_infrep3_inferred_cnvs.txt
56_infrep0_inferred_cnvs.txt
56_infrep4_inferred_cnvs.txt
56_infrep7_inferred_cnvs.txt
56_infrep1_tree_inferred.txt
56_infrep3_tree_inferred.txt
56_infrep8_inferred_cnvs.txt
57_infrep0_inferred_cnvs.txt
57_infrep7_inferred_cnvs.txt
57_infrep5_inferred_cnvs.txt
57_infrep2_inferred_cnvs.txt
57_infrep8_inferred_cnvs.txt
57_infrep1_inferred_cnvs.txt
57_infrep2_tree_inferred.txt
57_infrep6_tree_inferred.txt
57_infrep6_inferred_cnvs.txt
57_infrep3_tree_inferred.txt
57_infrep9_tree_inferred.txt
57_infrep0_tree_inferred.txt
57_infrep4_tree_inferred.txt
57_infrep3_inferred_cnvs.txt
57_infrep7_tree_inferred.txt
57_infrep1_tree_inferred.txt
57_infrep4_inferred_cnvs.txt
57_infrep9_inferred_cnvs.txt
57_infrep5_tree_inferred.txt
57_infrep8_tree_inferred.txt
58_infrep6_inferred_cnvs.txt
58_infrep9_tree_inferred.txt
58_infrep5_tree_inferred.txt
58_infrep4_tree_inferred.txt
58_infrep2_tre

81_infrep6_inferred_cnvs.txt
81_infrep5_inferred_cnvs.txt
81_infrep9_tree_inferred.txt
81_infrep1_inferred_cnvs.txt
81_infrep0_tree_inferred.txt
81_infrep8_tree_inferred.txt
81_infrep1_tree_inferred.txt
81_infrep3_tree_inferred.txt
81_infrep7_tree_inferred.txt
81_infrep3_inferred_cnvs.txt
81_infrep2_inferred_cnvs.txt
81_infrep4_tree_inferred.txt
81_infrep8_inferred_cnvs.txt
81_infrep5_tree_inferred.txt
81_infrep2_tree_inferred.txt
81_infrep4_inferred_cnvs.txt
81_infrep7_inferred_cnvs.txt
81_infrep9_inferred_cnvs.txt
82_infrep3_inferred_cnvs.txt
82_infrep2_tree_inferred.txt
82_infrep1_inferred_cnvs.txt
82_infrep1_tree_inferred.txt
82_infrep7_inferred_cnvs.txt
82_infrep4_inferred_cnvs.txt
82_infrep2_inferred_cnvs.txt
82_infrep6_tree_inferred.txt
82_infrep0_tree_inferred.txt
82_infrep7_tree_inferred.txt
82_infrep3_tree_inferred.txt
82_infrep5_tree_inferred.txt
82_infrep5_inferred_cnvs.txt
82_infrep6_inferred_cnvs.txt
82_infrep0_inferred_cnvs.txt
82_infrep9_inferred_cnvs.txt
82_infrep9_tre

14_infrep5_tree_inferred.txt
14_infrep1_tree_inferred.txt
14_infrep8_inferred_cnvs.txt
14_infrep0_inferred_cnvs.txt
14_infrep7_tree_inferred.txt
14_infrep9_tree_inferred.txt
14_infrep7_inferred_cnvs.txt
14_infrep8_tree_inferred.txt
14_infrep2_tree_inferred.txt
14_infrep5_inferred_cnvs.txt
14_infrep6_tree_inferred.txt
14_infrep4_tree_inferred.txt
14_infrep1_inferred_cnvs.txt
14_infrep0_tree_inferred.txt
14_infrep2_inferred_cnvs.txt
14_infrep3_inferred_cnvs.txt
14_infrep4_inferred_cnvs.txt
14_infrep3_tree_inferred.txt
14_infrep6_inferred_cnvs.txt
14_infrep9_inferred_cnvs.txt
15_infrep8_tree_inferred.txt
15_infrep2_tree_inferred.txt
15_infrep1_inferred_cnvs.txt
15_infrep9_tree_inferred.txt
15_infrep0_inferred_cnvs.txt
15_infrep5_tree_inferred.txt
15_infrep0_tree_inferred.txt
15_infrep7_inferred_cnvs.txt
15_infrep1_tree_inferred.txt
15_infrep8_inferred_cnvs.txt
15_infrep6_tree_inferred.txt
15_infrep7_tree_inferred.txt
15_infrep6_inferred_cnvs.txt
15_infrep4_tree_inferred.txt
15_infrep2_inf

38_infrep6_tree_inferred.txt
38_infrep7_tree_inferred.txt
38_infrep8_inferred_cnvs.txt
38_infrep5_inferred_cnvs.txt
38_infrep1_inferred_cnvs.txt
38_infrep5_tree_inferred.txt
38_infrep7_inferred_cnvs.txt
38_infrep2_tree_inferred.txt
38_infrep0_inferred_cnvs.txt
38_infrep3_tree_inferred.txt
38_infrep2_inferred_cnvs.txt
38_infrep4_inferred_cnvs.txt
38_infrep3_inferred_cnvs.txt
38_infrep1_tree_inferred.txt
38_infrep0_tree_inferred.txt
38_infrep8_tree_inferred.txt
38_infrep9_inferred_cnvs.txt
38_infrep4_tree_inferred.txt
40_infrep1_tree_inferred.txt
40_infrep5_tree_inferred.txt
40_infrep6_inferred_cnvs.txt
40_infrep3_tree_inferred.txt
40_infrep3_inferred_cnvs.txt
40_infrep5_inferred_cnvs.txt
40_infrep2_tree_inferred.txt
40_infrep7_tree_inferred.txt
40_infrep9_inferred_cnvs.txt
40_infrep4_tree_inferred.txt
40_infrep8_inferred_cnvs.txt
40_infrep8_tree_inferred.txt
40_infrep2_inferred_cnvs.txt
40_infrep0_tree_inferred.txt
40_infrep6_tree_inferred.txt
40_infrep1_inferred_cnvs.txt
40_infrep0_inf

62_infrep5_inferred_cnvs.txt
62_infrep1_inferred_cnvs.txt
62_infrep4_inferred_cnvs.txt
62_infrep3_tree_inferred.txt
62_infrep6_inferred_cnvs.txt
62_infrep8_inferred_cnvs.txt
62_infrep3_inferred_cnvs.txt
62_infrep7_inferred_cnvs.txt
62_infrep8_tree_inferred.txt
63_infrep2_tree_inferred.txt
63_infrep9_tree_inferred.txt
63_infrep5_inferred_cnvs.txt
63_infrep8_tree_inferred.txt
63_infrep6_inferred_cnvs.txt
63_infrep2_inferred_cnvs.txt
63_infrep7_inferred_cnvs.txt
63_infrep6_tree_inferred.txt
63_infrep7_tree_inferred.txt
63_infrep0_inferred_cnvs.txt
63_infrep4_tree_inferred.txt
63_infrep0_tree_inferred.txt
63_infrep4_inferred_cnvs.txt
63_infrep8_inferred_cnvs.txt
63_infrep1_inferred_cnvs.txt
63_infrep5_tree_inferred.txt
63_infrep3_inferred_cnvs.txt
63_infrep1_tree_inferred.txt
63_infrep3_tree_inferred.txt
63_infrep9_inferred_cnvs.txt
65_infrep8_tree_inferred.txt
65_infrep9_inferred_cnvs.txt
65_infrep5_tree_inferred.txt
65_infrep9_tree_inferred.txt
65_infrep3_inferred_cnvs.txt
65_infrep5_inf

92_infrep2_inferred_cnvs.txt
92_infrep0_tree_inferred.txt
92_infrep4_tree_inferred.txt
92_infrep6_inferred_cnvs.txt
92_infrep4_inferred_cnvs.txt
92_infrep5_inferred_cnvs.txt
92_infrep9_tree_inferred.txt
92_infrep0_inferred_cnvs.txt
92_infrep1_inferred_cnvs.txt
92_infrep3_tree_inferred.txt
92_infrep8_tree_inferred.txt
92_infrep5_tree_inferred.txt
92_infrep2_tree_inferred.txt
92_infrep8_inferred_cnvs.txt
92_infrep7_tree_inferred.txt
92_infrep3_inferred_cnvs.txt
92_infrep1_tree_inferred.txt
92_infrep6_tree_inferred.txt
95_infrep2_tree_inferred.txt
95_infrep9_tree_inferred.txt
95_infrep7_inferred_cnvs.txt
95_infrep8_tree_inferred.txt
95_infrep9_inferred_cnvs.txt
95_infrep3_inferred_cnvs.txt
95_infrep3_tree_inferred.txt
95_infrep4_inferred_cnvs.txt
95_infrep7_tree_inferred.txt
95_infrep1_inferred_cnvs.txt
95_infrep5_tree_inferred.txt
95_infrep6_inferred_cnvs.txt
95_infrep0_tree_inferred.txt
95_infrep8_inferred_cnvs.txt
95_infrep0_inferred_cnvs.txt
95_infrep1_tree_inferred.txt
95_infrep5_inf

In [27]:
scores_df.index[scores_df["10nodes_20regions_100000reads"]<=5]

Int64Index([], dtype='int64')

In [23]:
scores_df

10nodes_10regions_100000reads  10nodes_10regions_10000reads  \
0   8                              8                              
1   9                              7                              
2   7                              7                              
3   6                              9                              
4   9                              6                              
5   9                              8                              
6   7                              5                              
7   8                              6                              
8   6                              9                              
9   9                              6                              
10  8                              9                              
11  8                              6                              
12  8                              9                              
13  7                              8                              
14  8                              9                              
15  8                              9                              
16  8                              9                              
17  9                              7                              
18  7                              7                              
19  9                              8                              
20  8                              9                              
21  7                              7                              
22  9                              8                              
23  8                              9                              
24  6                              8                              
25  8                              8                              
26  8                              7                              
27  6                              9                              
28  8                              9                              
29  6                              9                              
30  6                              9                              
31  8                              7                              
32  9                              7                              
33  8                              6                              
34  9                              9                              
35  7                              9                              
36  8                              7                              
37  7                              8                              
38  8                              9                              
39  7                              7                              
40  8                              6                              
41  9                              6                              
42  6                              7                              
43  6                              7                              
44  9                              6                              
45  8                              9                              
46  8                              7                              
47  8                              9                              
48  9                              9                              
49  6                              9                              
50  7                              8                              
51  7                              8                              
52  7                              8                              
53  9                              8                              
54  9                              8                              
55  7                              7                              
56  9                              7                              
57  7                              7                              
58  6                              5               

In [ ]:
scores_df.head()

In [ ]:
5

In [ ]:
5

In [17]:
scores_df.mean()

10nodes_10regions_100000reads     7.95
10nodes_10regions_10000reads      0.86
10nodes_10regions_30000reads      5.26
10nodes_20regions_100000reads     8.10
10nodes_20regions_10000reads      1.63
10nodes_20regions_30000reads      6.01
10nodes_40regions_100000reads     8.03
10nodes_40regions_10000reads      2.84
10nodes_40regions_30000reads      6.21
20nodes_20regions_100000reads     8.37
20nodes_20regions_10000reads      0.00
20nodes_20regions_30000reads      1.38
20nodes_40regions_100000reads     7.87
20nodes_40regions_10000reads      0.09
20nodes_40regions_30000reads      2.02
20nodes_80regions_100000reads     7.77
20nodes_80regions_10000reads      0.12
20nodes_80regions_30000reads      2.45
30nodes_120regions_100000reads    7.44
30nodes_120regions_10000reads     0.18
30nodes_120regions_30000reads     1.48
30nodes_30regions_100000reads     8.33
30nodes_30regions_10000reads      0.02
30nodes_30regions_30000reads      2.21
30nodes_60regions_100000reads     7.99
30nodes_60regions_10000re

In [10]:
scores_df.mean()

10nodes_10regions_100000reads     7.37
10nodes_10regions_10000reads      7.26
10nodes_10regions_30000reads      7.48
10nodes_20regions_100000reads     7.80
10nodes_20regions_10000reads      7.85
10nodes_20regions_30000reads      7.94
10nodes_40regions_100000reads     4.62
10nodes_40regions_10000reads      4.85
10nodes_40regions_30000reads      4.64
20nodes_20regions_100000reads     8.89
20nodes_20regions_10000reads      8.90
20nodes_20regions_30000reads      8.90
20nodes_40regions_100000reads     8.13
20nodes_40regions_10000reads      8.07
20nodes_40regions_30000reads      8.14
20nodes_80regions_100000reads     3.92
20nodes_80regions_10000reads      4.37
20nodes_80regions_30000reads      4.07
30nodes_120regions_100000reads    2.98
30nodes_120regions_10000reads     2.69
30nodes_120regions_30000reads     3.51
30nodes_30regions_100000reads     8.91
30nodes_30regions_10000reads      8.68
30nodes_30regions_30000reads      8.92
30nodes_60regions_100000reads     8.09
30nodes_60regions_10000re

### Results after 512*n^2 iterations

The robustness score (the column on the right) less than 5 will be repeated this time with twice as much iterations.

`
10nodes_10regions_100000reads     7.37

10nodes_10regions_10000reads      7.26

10nodes_10regions_30000reads      7.48

10nodes_20regions_100000reads     7.80

10nodes_20regions_10000reads      7.85

10nodes_20regions_30000reads      7.94

10nodes_40regions_100000reads     4.62

10nodes_40regions_10000reads      4.85

10nodes_40regions_30000reads      4.64

20nodes_20regions_100000reads     8.89

20nodes_20regions_10000reads      8.90

20nodes_20regions_30000reads      8.90

20nodes_40regions_100000reads     8.13

20nodes_40regions_10000reads      8.07

20nodes_40regions_30000reads      8.14

20nodes_80regions_100000reads     3.92

20nodes_80regions_10000reads      4.37

20nodes_80regions_30000reads      4.07

30nodes_120regions_100000reads    2.98

30nodes_120regions_10000reads     2.69

30nodes_120regions_30000reads     3.51

30nodes_30regions_100000reads     8.91

30nodes_30regions_10000reads      8.68

30nodes_30regions_30000reads      8.92

30nodes_60regions_100000reads     8.09

30nodes_60regions_10000reads      7.71

30nodes_60regions_30000reads      8.09

dtype: float64 `